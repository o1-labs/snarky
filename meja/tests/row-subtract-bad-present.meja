let f = fun (x : [[`A | `B] - `A], y : [`A | `B], b) => {
  if (b) { x; } else { y; };
};
