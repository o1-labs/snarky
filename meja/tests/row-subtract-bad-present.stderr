File "tests/row-subtract-bad-present.meja", line 2, characters 23-24:
Error: Incompatible types [[`A | `B] - `A] and [`A | `B]:
       Cannot unify [[`A | `B] - `A] and [`A | `B]:
       the constructor `A is not present in both.
